README.md

```diff
@@ -52,32 +52,54 @@ COOL certificates.
-| assume_dns_certificatesbucketfullaccess_policy_name | The name to assign the IAM policy that allows assumption of the role that allows full access to the certificates bucket in the DNS account. | `string` | `DNS-AssumeCertificatesBucketFullAccess` | no |
-| assume_dns_route53resourcechange_cyber_dhs_gov_policy_description | The description to associate with the IAM policy that allows assumption of the role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone in the DNS account. | `string` | `The IAM policy that allows assumption of the role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone in the DNS account.` | no |
-| assume_dns_route53resourcechange_cyber_dhs_gov_policy_name | The name to assign the IAM policy that allows assumption of the role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone in the DNS account. | `string` | `DNS-AssumeRoute53ResourceChange-cyber.dhs.gov` | no |
-| aws_region | The AWS region to deploy into (e.g. us-east-1) | `string` | `us-east-1` | no |
-| certificate_managers_group_name | The name of the group to be created for certificate manager users. | `string` | `certificate_managers` | no |
+| assume\_dns\_certificatesbucketfullaccess\_policy\_description | The description to associate with the IAM policy that allows assumption of the role that allows full access to the certificates bucket in the DNS account. | `string` | `"The IAM policy that allows assumption of the role that allows full access to the certificates bucket in the DNS account."` | no |
+| assume\_dns\_certificatesbucketfullaccess\_policy\_name | The name to assign the IAM policy that allows assumption of the role that allows full access to the certificates bucket in the DNS account. | `string` | `"DNS-AssumeCertificatesBucketFullAccess"` | no |
+| assume\_dns\_route53resourcechange\_cyber\_dhs\_gov\_policy\_description | The description to associate with the IAM policy that allows assumption of the role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone in the DNS account. | `string` | `"The IAM policy that allows assumption of the role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone in the DNS account."` | no |
+| assume\_dns\_route53resourcechange\_cyber\_dhs\_gov\_policy\_name | The name to assign the IAM policy that allows assumption of the role that allows sufficient permissions to modify resource records in the cyber.dhs.gov zone in the DNS account. | `string` | `"DNS-AssumeRoute53ResourceChange-cyber.dhs.gov"` | no |
+| aws\_region | The AWS region to deploy into (e.g. us-east-1) | `string` | `"us-east-1"` | no |
+| certificate\_managers\_group\_name | The name of the group to be created for certificate manager users. | `string` | `"certificate_managers"` | no |
 | tags | Tags to apply to all AWS resources created | `map(string)` | `{}` | no |
```

Comment:
The variable description would need to be updated as well.

```suggestion
| tags | Tags to apply to all AWS resources created. | `map(string)` | `{}` | no |
```

Reply:
See commit dc20303.